src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -682,6 +682,13 @@ def load_arguments(self, _):
         c.argument('acr', validator=validate_registry_name)
         c.argument('node_name')
 
+    with self.argument_context('aks machine') as c:
+        c.argument('cluster_name', help='The cluster name.')
+        c.argument('nodepool_name', validator=validate_nodepool_name, help='The node pool name.')
+
+    with self.argument_context('aks machine show') as c:
+        c.argument('machine_name', help='to display specific information for all machines.')
```

Comment:
This help information seems to be inaccurate, it doesn't look like it's describing the parameter of `--machine-name`